modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd

```diff
@@ -397,12 +358,14 @@ library(janitor)
 
 ## janitor `clean_names`
 
-The `clean_names` function can intuit what fixes you might need. Here it makes everything consistent.
+The `clean_names` function can intuit what fixes you might need. Here it make sure year names aren't just a number, so that the colnames dont need ticks or quotes to be used.
 
 ```{r}
-head(annualDosage, 2)
-clean_AD <- clean_names(annualDosage)
-head(clean_AD, 2)
+CO2 <- 
+  read_csv("https://daseh.org/data/Yearly_CO2_Emissions_1000_tonnes.csv")
```

Comment:
why not use dasehr::yearly_co2_emissions ?

Reply:
you mean that you don't want me to read it in from the csv file? cause that is the dataset I used